pyproject.toml

```diff
@@ -172,3 +161,60 @@ force-sort-within-sections = true
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.pixi.workspace]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tool.pixi.environments]
+default = { features = [], solve-group = "default" }
+dev = { features = ["dev"], solve-group = "default" }
+test = { features = ["test", "dev"], solve-group = "default" }
+
+py311 = { features = ["py311"], solve-group = "py311" }
+py311-dev = { features = ["dev", "py311"], solve-group = "py311" }
+py311-test = { features = ["test", "dev", "py311"], solve-group = "py311" }
+
+py312 = { features = ["py312"], solve-group = "py312" }
+py312-dev = { features = ["dev", "py312"], solve-group = "py312" }
+py312-test = { features = ["test", "dev", "py312"], solve-group = "py312" }
+
+py313 = { features = ["py313"], solve-group = "py313" }
+py313-dev = { features = ["dev", "py313"], solve-group = "py313" }
+py313-test = { features = ["test", "dev", "py313"], solve-group = "py313" }
+
+[tool.pixi.dependencies]
+cython = ">=3.0.12,<4"
+libmo_unpack = ">=3.1.2,<4"
+numpy = ">=1.25.0,<3"
+setuptools = ">=79.0.1,<80"
+setuptools-scm = ">=8.3.1,<9"
+
+[tool.pixi.feature.dev.dependencies]
+pre-commit = ">=4.2.0,<5"
+ruff = ">=0.11.6,<0.12"
+
+[tool.pixi.feature.dev.pypi-dependencies]
+mo-pack = { path = ".", editable = true }
+
+[tool.pixi.feature.dev.tasks]
+build = { cmd = ["python", "setup.py", "build_ext", "--inplace"], depends-on = ["clean"] }
+clean = { cmd = ["python", "setup.py", "clean_cython"] }
+fmt = "ruff format"
+lint = "ruff check"
+pytest = { cmd = ["pytest"], depends-on = ["build"] }
```

Comment:
We could pass through command line arguments here to `pytest` for added contributor convenience ...